apis/cc-infix-en_GB/atrium-api-cc-infix-en_GB-jvm/src/test/kotlin/ch/tutteli/atrium/api/cc/infix/en_GB/CharSequenceContainsRegexAssertionsSpec.kt

```diff
@@ -68,9 +65,9 @@ class CharSequenceContainsRegexAssertionsSpec : ch.tutteli.atrium.spec.integrati
 
         private fun containsShortcut(plant: Assert<CharSequence>, a: String, aX: Array<out String>): Assert<CharSequence> {
             return if (aX.isEmpty()) {
-                plant.asExpect().containsRegex(a).asAssert()
+                plant containsRegex a
```

Comment:
what's wrong with the replacement?

Reply:
Nothing's wrong I am just try out what makes the test failed.

Reply:
I see, I can give you a few hints as some of them are hard to spot:
- there was actually a mistake in the new api-infix. The function is not called `contains` for RegexPatterns` but `containsRegex`  -- I will fix this with https://github.com/robstoll/atrium/pull/472, please rebase once it it merged into master. 

The rest is harder to spot:
- wherever we use helper objects such as `Values` in api-cc-infix, we need to use the corresponding helper functino in api-infix - I did not think about either. I will leave a review comment with an example at a corresponding place
- some error messages have changed in api-infix, which means we need to change the expectation in api-cc-infix as well once we switch. I will also leave a review comment